create_deb.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+PROGRAM_NAME=sdlhmi
+VERSION=6.2.0
+PROGRAM_NAME=$PROGRAM_NAME-$VERSION
+
+rm -rf $PROGRAM_NAME
+mkdir $PROGRAM_NAME
+cp -r  deb/* $PROGRAM_NAME/
+mkdir -p $PROGRAM_NAME/usr/bin/
+mkdir -p $PROGRAM_NAME/usr/share/sdlhmi
+mkdir -p $PROGRAM_NAME/usr/share/doc/sdlhmi
+cp sdlhmi $PROGRAM_NAME/usr/bin
+
+for entity in $(ls);do
+  cp -rv $entity $PROGRAM_NAME/usr/share/sdlhmi/
```

Comment:
@LuxoftAKutsan is this going to be executed from root user?

Reply:
No, this will be executed from regular user